frontend/.env

```diff
@@ -1 +1,4 @@
-VITE_BACKEND_URL="http://localhost:3000"
+# Not suggested modifying the backend port
+VITE_BACKEND_URL="http://localhost:8080"
+# Add your google client ID here
+VITE_GOOGLE_CLIENT_ID=""
```

Comment:
Didn't you need client secret while implementing this way? Asking because can't see any Client Secret environment variable  used.

Reply:
no, i didt require any client secrets in this approach, i had used another approach which required client secrets, but this approach doesnt require client secrets.